packages/dui3/lib/common/generated/gql/graphql.ts

```diff
@@ -209,7 +209,7 @@ export type Comment = {
    * Legacy comment viewer data field
    * @deprecated Use the new viewerState field instead
    */
-  data?: Maybe<LegacyCommentViewerData>;
+  data?: Maybe<Scalars['JSONObject']>;
```

Comment:
@fabis94 Is it normal that these lines in `packages/dui3/lib/common/generated/gql/graphql.ts` changed automatically? I haven't changed them.

Reply:
yes this is normal. generally you can ignore all generated files.